packages/frontend-2/pages/downloads.vue

```diff
@@ -1,21 +1,21 @@
 <template>
   <div>
     <Portal to="navigation">
-      <HeaderNavLink :to="'/downloads'" name="Speckle Connectors"></HeaderNavLink>
+      <HeaderNavLink :to="'/downloads'" name="Speckle connectors"></HeaderNavLink>
     </Portal>
     <div
       class="flex flex-col md:flex-row space-y-2 space-x-2 justify-between mb-4 md:items-center"
     >
       <div>
-        <h5 class="h4 font-bold">Connector Downloads</h5>
+        <h5 class="h4 font-bold">Connector downloads</h5>
         <div class="text-sm text-foreground-2 max-w- max-w-sm">
           Most of our connectors are available through Speckle Manager. You can also
           direct download the individual installers below.
         </div>
       </div>
       <div>
         <FormButton size="lg" full-width @click="showManagerDownloadDialog = true">
-          Download Manager
+          Download manager
```

Comment:
```suggestion
          Download Manager
```
Should be uppercase. Manager is a product name, like "Speckle" or "Revit".

Reply:
Got it! Updated :)